test/relations.test.js

```diff
@@ -2453,9 +2621,17 @@ describe('relations', function() {
       if (!author) return done();
       Author.findById(author.id, function(err, author) {
         author.pictures(function(err, pics) {
-          pics.should.have.length(2);
```

Comment:
Why is Cassandra returning a different number of scoped items than other connectors? I find that very suspicious! Does it mean the scopes actually don't work with Cassandra?

It will be probably best to skip this entire test for Cassandra.